runtime/onert/core/src/compiler/Compiler.cc

```diff
@@ -105,27 +105,24 @@ void ManualSchedulerOptions::setBackendMap(const ir::Model &model, const std::st
     const auto &key_str = key_val.at(0);
     const auto &val = key_val.at(1);
     auto key = static_cast<uint32_t>(std::stoi(key_str));
-
-    model.at(ir::SubgraphIndex{0})
-      ->operations()
-      .at(ir::OperationIndex{key}); // Check if exist, or this wil throw
```

Comment:
Please see https://github.com/Samsung/ONE/pull/9334#discussion_r905917895. The same check is done before `schedule`.